src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java

```diff
@@ -63,6 +63,28 @@ public void basicMarbleRoll(final String dataSourcesPath,
         runCommandLine(arguments);
     }
 
+//    @Test()
```

Comment:
Can we remove commented out lines?

Reply:
If you want to disable a test (temporarily) there is a flag in the `@Test(....)` annotation.  Maybe `enabled=false` ?

Reply:
I added the enabled=false flag so this can be used later.